test/converters/test_converters.py

```diff
@@ -23,23 +23,27 @@
 from qiskit_optimization import QuadraticProgram, QiskitOptimizationError
 from qiskit_optimization.algorithms import MinimumEigenOptimizer, CplexOptimizer, ADMMOptimizer
 from qiskit_optimization.algorithms.admm_optimizer import ADMMParameters
-from qiskit_optimization.converters import (InequalityToEquality, IntegerToBinary,
-                                            LinearEqualityToPenalty, SpecialConstraintToPenalty)
+from qiskit_optimization.converters import (
+    InequalityToEquality,
+    IntegerToBinary,
+    LinearEqualityToPenalty,
+    LinearInequalityToPenalty,
+)
 from qiskit_optimization.problems import Constraint, Variable
 
 logger = logging.getLogger(__name__)
 
 QUBIT_OP_MAXIMIZE_SAMPLE = (
```

Comment:
You can simplify it by using `PauliSumOp.from_list`.
https://qiskit.org/documentation/stubs/qiskit.opflow.primitive_ops.PauliSumOp.html?highlight=paulisumop#qiskit.opflow.primitive_ops.PauliSumOp.from_list

Reply:
We discussed and concluded it should be done in another PR.